etc/browser/lib/crypto.js

```diff
@@ -12,6 +12,9 @@
  *
  */
 
+var buffer = require('buffer');
+var Buffer = buffer.Buffer;
```

Comment:
@mtth I left this /etc file in, I wasn't sure if it was a polyfill for the browser and may need the explicit import.

Reply:
Good call, it is. I'd forgotten about that one.